doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -562,16 +403,159 @@ You can remove a placement group at any time to free its allocated resources.
+- **Ray Dashboard** is a UI tool to inspect placement group states.
+- **Ray State API** is a CLI to inspect placement group states.
+
+.. tabbed:: Ray Status
+
+  The CLI command ``ray status`` provides the autoscaling status of the cluster. 
+  It provides the "resource demands" from unscheduled placement groups as well as the resource reservation status.
+
+  SANG-TODO images
+
+.. tabbed:: Dashboard
+
+  :ref:`The dashboard job view <dash-jobs-view>` provides the placement group table that displays the scheduling state and metadata of the placement group.
+
+  .. note::
+
+    Ray dashboard is only available when Ray is installed with ``pip install "ray[default]"``.
+
+.. tabbed:: Ray State API
+
+  :ref:`Ray state API <state-api-overview-ref>` is a CLI tool to inspect the state of Ray resources (tasks, actors, placement groups, etc.). 
+
+  ``ray list placement-groups`` provides the metadata and the scheduling state of the placement group.
+  ``ray list placement-groups --detail`` provides stats and scheduling state in a greater detail.
+
+  .. note::
+
+    State API is only available when Ray is installed with ``pip install "ray[default]"``
+
+Inspect Placement Group Scheduling State
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+From the above tools, you can see the state of the placement group. The definition of states are specified in the following files.
+
+- `High level state <https://github.com/ray-project/ray/blob/03a9d2166988b16b7cbf51dac0e6e586455b28d8/src/ray/protobuf/gcs.proto#L579>`_
+- `Details <https://github.com/ray-project/ray/blob/03a9d2166988b16b7cbf51dac0e6e586455b28d8/src/ray/protobuf/gcs.proto#L524>`_
+
+.. _pgroup-strategy:
+
+[Advanced] Placment Strategy
+----------------------------
+
+Often, you'd like to reserve bundles with placement constraints. For example, you'd like to pack your bundles to the same
+node or spread out to multiple nodes as much as possible.
+
+Ray currently supports the following placement group strategies. The default scheduling policy is ``PACK``:
+
+**STRICT_PACK**
+
+All bundles must be placed into a single node on the cluster. It is useful when you want to maximize the locality.
+
+**PACK**
+
+All provided bundles are packed onto a single node on a best-effort basis.
+If strict packing is not feasible (i.e., some bundles do not fit on the node), bundles can be placed onto other nodes nodes.
+
+**STRICT_SPREAD**
+
+Each bundle must be scheduled in a separate node.
+
+**SPREAD**
+
+Each bundle will be spread onto separate nodes on a best effort basis.
+If strict spreading is not feasible, bundles can be placed overlapping nodes.
+
+.. tabbed:: Python
+
+    .. code-block:: python
+
+      # Reserve a placement group of 2 bundles
+      # that have to be packed on the same node.
+      pg = placement_group([{"CPU": 1}, {"GPU": 1}], strategy="STRICT_PACK")
+
+[Advanced] Nested Placement Groups
```

Comment:
I don't think this is nested placement group: to me nested placement group means you create a child pg using the resources reserved by the parent pg.

Reply:
any good recommendatino for the title?

Reply:
I changed it